shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -434,15 +434,17 @@ private void doInitialFlutterViewRun() {
   private String maybeGetInitialRouteFromIntent(Intent intent) {
     if (host.shouldHandleDeeplinking()) {
       Uri data = intent.getData();
-      if (data != null && !data.getPath().isEmpty()) {
+      if (data != null) {
```

Comment:
`FlutterActivityAndFragmentDelegateTest.java` has a few examples about how to test this.